### PR TITLE
Preprod: movie: Red Dwarf:The Promised Land (2020) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
I want to say "New century, same old nonsense", but Doug Naylor stepped up his game a bit for this one. Rating 3/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 1]
```diff
@@ -197,6 +197,53 @@ powered exoskeleton [Dalton hooked up to his spine a robotic apparatus with long
 programmable person [Ray could be programmed to do Rising Spirit Tech corporation's bidding.]
 
 
+movie: Red Dwarf:The Promised Land (2020)
+=========================================
+
+:: Title
+Red Dwarf:The Promised Land
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2020s.st.txt:204]: Missing space after colon?

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 2]
```diff
@@ -197,6 +197,53 @@ powered exoskeleton [Dalton hooked up to his spine a robotic apparatus with long
...
+pledged to annihilate all those who don't worship him as their god.
+
+:: Date
+2020-04-09
+
+:: References
+https://en.wikipedia.org/wiki/Red_Dwarf:_The_Promised_Land
+
+:: Ratings
+3 <paul>
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2020s.st.txt:222]: Typo?

**Review by odinlake — APPROVED** [review]

File: notes/stories/film/film-scifi-2020s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt:204]: Indeed. Fixed.

File: notes/stories/film/film-scifi-2020s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt:222]: You'd think so, wouldn't you? But go figure it actually had a coherent plot.

**Pull request merged**